.gitignore

```diff
@@ -14,3 +14,10 @@
 # Ignore all logfiles and tempfiles.
 /log/*.log
 /tmp
+
+# Ignore npm modules for Grunt
+node_modules
+
+# Ignore Gruntfiles.js
+Gruntfile.js
+mod
```

Comment:
Did we install grunt while I was on vacation last week? If so this is great, otherwise we probably need to remove these from this ignore file.